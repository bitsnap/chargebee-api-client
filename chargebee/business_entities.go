package chargebee

//func ListBusinessEntityTransfersPageSortBy(site string, offset string, sortBy *SortBy) ([]BusinessEntityTransfer, string, error) {
//
//}
//
//func ListBusinessEntityTransfersPage(site string, offset string) ([]BusinessEntityTransfer, string, error) {
//	return ListBusinessEntityTransfersPageSortBy(site, offset, nil)
//}
