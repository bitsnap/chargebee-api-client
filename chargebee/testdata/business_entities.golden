# Generated by goldga. DO NOT EDIT.
[snapshots]
"Chargebee client test when accessing business entities API should be able list business entity transfers" = '''
([]chargebee.BusinessEntityTransfer) {
}
'''
